### PR TITLE
Update index.html.md

### PR DESCRIPTION
**Pull request by Galser**
We had several customers stepping on S3 test problem in ReplicatedUI dashboard during config phase , perhaps it should be outlined  here, in our manual.

The source of the information is here : https://help.replicated.com/docs/config-screen/test-procs/#aws-auth

> Feel free to decline if you feel that this is more suited for KB artilce in Support

### CODE REVIEW

**Review by nervda — CHANGES_REQUESTED** [review]
**Note:** During the interactive installation Replicated (TFE setup dashboard) can execute S3 access tests that will be performed using `ListAllMyBuckets` action. And, as a result - require additional permission added *temporarily* to the policy above - **ListAllMyBuckets**. Later, in operations Terraform Enterprise not require this permission.

File: content/source/docs/enterprise/before-installing/index.html.md [hunk 1]
```diff
@@ -163,6 +163,8 @@ At a minimum, Terraform Enterprise requires the following S3 permissions:
     ]
 }
 ```
+-> **Note:** During the interactive installation Replicated (TFE setup dashboard) can execute S3 access tests that will be perfomed using `ListAllMyBuckets` action. And, as a result - require additional permission added *temporary* to the policy above - **ListAllMyBuckets**. Later, in operations Terraform Enterprise not require this permission.
```
Comment (sudomateo) [inline content/source/docs/enterprise/before-installing/index.html.md:166]: We can be a bit more brief here.

```suggestion
-> **Note:** The `s3:ListAllMyBuckets` permission is necessary when testing authentication via the Replicated web console. However, the permission is not required for Terraform Enterprise to function and can be removed once the authentication is successfully tested.
```

**Review by sudomateo — CHANGES_REQUESTED** [review]
Thank you for working on this! Submitted a suggestion.

**Comment by sudomateo** [comment]
@Galser This should not have been merged without an approval. Just an FYI for next time.

**Pull request merged**